community/examples/gke-tpu-v4/README.md

```diff
@@ -0,0 +1,6 @@
+# GKE TPU V4 blueprint
+
+This example shows how TPU v4 cluster can be created and be used to run a job that requires TPU capacity on GKE.
+
+<to-be-added>
```

Comment:
Anything in specific to be captured here?

Reply:
Let's link to public documentation that provides customers with the basic knowledge of TPU required to tweak the blueprint and solve for their use case.

Reply:
https://cloud.google.com/tpu/docs/v4 is one reference link I have on mind for this v4 blueprint.